### PR TITLE
adding mobile menu to Portfolio page.

### PR DESCRIPTION
**Pull request by jossi30**
adding a mobile menu that opens and closes by clicking the nav bar and nav links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,34 +12,30 @@
...
-          d="M4 6h16M4 12h16M4 18h16"
-        ></path>
-      </svg>
-      <div class="nav-links">
-        <ul>
-          <li class="portfolio"><a href="#works">Portfolio</a></li>
-          <li class="portfolio"><a href="#about-myself">About</a></li>
-          <li class="portfolio"><a href="#contact-me">Contact</a></li>
+      <nav class="navbar">
+        <a href="#" class="logo">jossi30</a>
```
Comment (tufoinnkuo10) [inline index.html:16]: - [ ] great job so far, everything works fine. please ensure that the social icons are not showing on the menu popup so as to follow the figma design. great job.
Screenshot (507)_LI

File: index.html [hunk 2]
```diff
@@ -12,34 +12,30 @@
...
+        <div class="menu-toggle" id="mobile-menu">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul class="nav-menu">
```
Comment (tufoinnkuo10) [inline index.html:22]: - [ ] kindly ensure that when the menu comes up, it should not be posible to scroll through your portfolio unless using the menu links or when the menu is closed.
Screenshot (508)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @jossi30 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 3]
```diff
@@ -11,9 +11,28 @@
...
+        alt=""
+      />
+    </div>
+
+    <ul class="mobile-menu-items">
+      <li>
+        <a onclick="myFunction()" href="#portfolio-id">Portoflio</a>
+      </li>
+      <li><a onclick="myFunction()" href="#about-id">About</a></li>
+      <li><a onclick="myFunction()" href="#contact-me-id">Contact</a></li>
```
Comment (ShahierNashaat) [inline index.html:30]: - [ ] Kindly write `href` values correct so the links work properly.

```html
<li><a onclick="myFunction()" href="#works">Portoflio</a></li>
<li><a onclick="myFunction()" href="#about-myself">About</a></li>
<li><a onclick="myFunction()" href="#contact-me">Contact</a></li>
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @jossi30, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**